src/uikit.cs

```diff
@@ -5235,7 +5235,7 @@ interface UIDocument : NSFilePresenter, NSProgressReporting {
 
 		[iOS (8,0)]
 		[Export ("restoreUserActivityState:")]
-		void RestoreUserActivityState (NSUserActivity userActivity);
+		new void RestoreUserActivityState (NSUserActivity userActivity);
```

Comment:
Can you try locally to remove this method, so that we get the one from the protocol, and see what happens to the api diff? I _think_ it won't be a breaking change, in which case it's better to remove it.